### PR TITLE
NMS-14145: Allow to disable DCB scheduling

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://issues.opennms.org/browse/NMS-14145

### CODE REVIEW

**Review by synqotik — DISMISSED** [review]
LGTM

**Review by cgorantla — APPROVED** [review]
LGTM, we should probably update doc as well.

File: features/device-config/monitor/src/main/java/org/opennms/features/deviceconfig/monitors/DeviceConfigMonitor.java [hunk 1]
```diff
@@ -140,10 +142,17 @@ public PollStatus poll(MonitoredService svc, Map<String, Object> parameters) {
 
         final Date lastRun = new Date(getKeyedLong(parameters, LAST_RETRIEVAL, 0L));
         final String cronSchedule = getKeyedString(parameters, DeviceConfigConstants.SCHEDULE, DeviceConfigConstants.DEFAULT_CRON_SCHEDULE);
```
Comment (fooker) [inline features/device-config/monitor/src/main/java/org/opennms/features/deviceconfig/monitors/DeviceConfigMonitor.java:144]: Should `never` be the default?

**Review by fooker — APPROVED** [review]

File: features/device-config/monitor/src/main/java/org/opennms/features/deviceconfig/monitors/DeviceConfigMonitor.java (hunk 1 above)
Comment (christianpape) [reply features/device-config/monitor/src/main/java/org/opennms/features/deviceconfig/monitors/DeviceConfigMonitor.java:144]: Good question, but nothing that I want to decide in this PR.

File: docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc [hunk 2]
```diff
@@ -22,4 +22,6 @@ If you want to enable device config backup on existing nodes in a requisition, s
 .. `schedule`: a cron expression to specify the time to fetch the device backup configuration (default is `0 0 0 * * ?`, every 24 hrs at 12:00 a.m.)
 . Click *Save* to save the metadata and then *Save* to save the requisition.
 . Click *Return* and click *Synchronize*.
-. Choose a scan option and click *Synchronize*.
\ No newline at end of file
+. Choose a scan option and click *Synchronize*.
+
+NOTE: By specifying the value `never` for the key `schedule` device configuration backups will not by triggered automatically. Of course it is still possible to retrieve backups on demand by using the Web-UI or the ReST services.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc:27]: NOTE: By specifying the value `never` for the key `schedule`, device configuration backups will not by triggered automatically. It is still possible to retrieve backups on demand by using the Web UI or the REST services.

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
minor edits to the docs. Thanks for adding note!

File: docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc (hunk 2 above)
Comment (christianpape) [reply docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc]: done.

File: docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc [hunk 3]
```diff
@@ -24,4 +24,4 @@ If you want to enable device config backup on existing nodes in a requisition, s
 . Click *Return* and click *Synchronize*.
 . Choose a scan option and click *Synchronize*.
 
-NOTE: By specifying the value `never` for the key `schedule` device configuration backups will not by triggered automatically. Of course it is still possible to retrieve backups on demand by using the Web-UI or the ReST services.
+NOTE: By specifying the value `never` for the key `schedule` device configuration backups will not be triggered automatically. It is still possible to retrieve backups on demand by using the Web-UI or the REST services.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc:27]: Sorry, I don't know why it's not letting me quote the line directly. Please add comma after "schedule" and remove the hyphen in "Web-UI". 

NOTE: By specifying the value `never` for the key `schedule`, device configuration backups will not be triggered automatically. It is still possible to retrieve backups on demand by using the Web UI or the REST services.

**Review by Bonrob2 — CHANGES_REQUESTED** [review]

File: docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc (hunk 3 above)
Comment (christianpape) [reply docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc]: Sorry, I missed that. Should be fine now.

**Pull request merged**